distribution/docs/src/main/jdocs/content/_configuring/configuring-managed-service-factory-bundles-contents.adoc

```diff
@@ -31,97 +31,143 @@ The following table shows the format to use for each property type supported.
-|name=B["true\|false","true\|false",…​]
-|`authorizedList=B["true","false"]`
-
-|Lists of Integers
-|name=I["value1","value2",…​]
-|`sizes=I["10","20","30"]`
-
-|Lists of Longs
-|name=L["value1","value2",…​]
-|`sizes=L["100","200","300"]`
-
-|Lists of Floats
-|name=F["value1","value2",…​]
-|`sizes=F["0x3f8ccccd","0x400ccccd","0x40533333"]`
-
-|Lists of Doubles
-|name=D["value1","value2",…​]
-|`sizes=D["0x3f8fbe77","0x400ef9db","0x4056147b"]`
-
-|Vectors of Strings
-|name=("value1","value2",…​)
-|`names=("A","B","C")`
-
-|Vectors of Booleans
-|name=B("true\|false","true\|false",…​)
-|`authorizedVector=B("true","false")`
-
-|Vectors of Integers
-|name=I("value1","value2",…​)
-|`sizes=I("10","20","30")`
-
-|Vectors of Longs
-|name=L("value1","value2",…​)
-|`sizes=L("100","200","300")`
-
-|Vectors of Floats
-|name=F("value1","value2",…​)
-|`sizes=F("0x3f8ccccd","0x400ccccd","0x40533333")`
-
-|Vectors of Doubles
-|name=D("value1","value2",…​)
-|`sizes=D("0x3f8fbe77","0x400ef9db","0x4056147b")`
+|String
+a|name = "value"
+a|`name = "John"`
+
+|Boolean
+a|name = B"true\|false" +
+name = b"true\|false"
+a|`authorized = B"true"`
+
+|Integer
+a|name = I"value" +
+name = i"value"
+a|`timeout = I"10"`
+
+|Long
+a|name = L"value" +
+name = l"value"
+a|`diameter = L"100"`
+
+|Float
+a|name = F"value" +
+name = f"value"
+a|`cost = F"1065353216"`
+
+|Double
+a|name = D"value" +
+name = d"value"
+a|`latitude = D"4636745974857667812"`
+
+|List of Strings
+a|name = ["value1","value2",…​]
+a|```
+complexStringArray = [ \
+  "{\"url\"\ \"http://test.sample.com\"\ \"layers\"\ [\"0\"]\ \"VERSION\"\ \"1.1\|1.2\"\ \"image/png\"}\ \"beta\"\ 1}", \
+  "{\"url\"\ \"http://test.sample.com"\ 0.5}","/solr\=SAML\|PKI\|basic", \
+  "/security-config\=SAML\|basic", \
+  ]
+```
+
+|List of Booleans
+a|name = B["true\|false","true\|false",…​] +
+name = b["true\|false","true\|false",…​]
+a|```
+authorizedList = B[ \
+  "true", \
+  "false", \
+  ]
+```
+
+|List of Integers
+a|name = I["value1","value2",…​] +
+name = i["value1","value2",…​]
+a|```
+sizes = I[ \
+  "10", \
+  "20", \
+  "30", \
+  ]
+```
+
+|List of Longs
+a|name = L["value1","value2",…​] +
+name = l["value1","value2",…​]
+a|```
+sizes = L[ \
+  "100", \
+  "200", \
+  "300", \
+  ]
+```
+
+|List of Floats
+a|name = F["value1","value2",…​] +
+name = f["value1","value2",…​]
+a|```
+sizes = F[ \
+  "1066192077", \
+  "1074580685", \
+  "1079194419", \
+  ]
+```
+
+|List of Doubles
+a|name = D["value1","value2",…​] +
+name = d["value1","value2",…​]
+a|```
+sizes = D[ \
+  "4607736361554183979", \
+  "4612212939583790252", \
+  "4614714689176794563", \
+  ]
+```
 
 |===
 
 [NOTE]
 ====
-* Lists and Vectors of values can be prefixed with any of the supported types (`B`, `I`, `L`, `F` or `D`).
-* To prevent any configuration issues, the `=` signs used in values should be escaped using a backslash (`\`).
+* Lists of values can be prefixed with any of the supported types (`B`, `b`, `I`, `i`, `L`, `l`, `F`, `f`, `D`, or `d`).
```

Comment:
Should we remove float and double from this list? We removed them from another list.